docs/node-configuration.md

```diff
@@ -403,3 +403,25 @@ where:
 
   Note that `Transaction` is a NeoGo extension that isn't supported by the NeoC#
   node and must be disabled on the public Neo N3 networks.
+
+## DB compatibility
```

Comment:
Needs the same indent as above.

Reply:
We have a list above and this is a new section, but I've improved the list a bit anyway.